client/ayon_harmony/plugins/create/create_render_layers.py

```diff
@@ -0,0 +1,1005 @@
+        mark_for_review = pre_create_data.get("mark_for_review")
+        if mark_for_review is None:
+            mark_for_review = self.mark_for_review
+        creator_attributes["mark_for_review"] = mark_for_review
+        creator_attributes["render_target"] = pre_create_data["render_target"]
+        creator_attributes["render_layer_instance_id"] = render_layer_instance_id
+
+        instance_data["layer_name"] = marked_layer_name
+
+        node = self._create_node_for_pass(layer, product_name, self.rename_read)
+        self.log.info(f"Created node:: {node}")
+        return node
+
+    def _get_selected_layers(self):
+        return {layer for layer in get_layers_info() if layer["selected"]}
+    
+    def _create_node_for_pass(self, layer, product_name, rename_read):
+        self_name = self.__class__.__name__
+        layer_name = layer["name"]
+        
+        created_node = harmony.send(
+            {
+                "function": f"AyonHarmony.Creators.{self_name}.createPassNode",
+                "args": [layer_name, product_name, rename_read]
+            }
+        )["result"]
+
+        return created_node
+    
+    def _get_used_layer(self, marked_layer_name, layers_data):
+        if marked_layer_name is not None:
+            layers_by_name = {layer["name"]: layer for layer in layers_data}
+            layer = layers_by_name.get(marked_layer_name)
+            if layer is None:
+                raise CreatorError(
+                    f"Layer with name \"{marked_layer_name}\" was not found")
+        else:
+            marked_layers = [
+                layer
+                for layer in layers_data
+                if layer["selected"]
+            ]
+
+            # Raise if nothing is selected
+            if not marked_layers:
+                raise CreatorError(
+                    "Nothing is selected. Please select layers.")
+            if len(marked_layers) != 1:
+                raise CreatorError(
+                    "Select just 1 layer.")
+            layer = marked_layers[0]
+
+        return layer
+
+    def get_dynamic_data(
+        self,
+        project_name,
+        folder_entity,
+        task_entity,
+        variant,
+        host_name,
+        instance
+    ):
+        dynamic_data = super().get_dynamic_data(
+            project_name,
+            folder_entity,
+            task_entity,
+            variant,
+            host_name,
+            instance
+        )
+        dynamic_data["renderpass"] = "{renderpass}"
+        dynamic_data["renderlayer"] = "{renderlayer}"
+        if "data" in instance: # passed from Auto creator
+            renderpass = instance["data"].get("renderpass")
+            if renderpass:
+                dynamic_data["renderpass"] = renderpass
+            renderlayer = instance["data"].get("renderlayer")
+            if renderlayer:
+                dynamic_data["renderlayer"] = renderlayer
```

Comment:
This would fill the product name and you would not be able to re-fill it when render layer variant changes (for example).
```suggestion
```